src/System.IO.Compression.Brotli/System/IO/Compression/Brotli.cs

```diff
@@ -117,6 +117,10 @@ internal static void EnsureInitialized(ref State state, bool compress)
 
         public static int GetMaximumCompressedSize(int inputSize)
         {
+            if (inputSize < 0 || inputSize > 2147483132) // 2^32 - 1 - 515 (max compressed extra bytes)
```

Comment:
It should be a const with the comment as well, imo.

What would the math operation return if inputSize > 2147483132 (i.e., do we even need this check)?